src/GeneralStateTestsFiller/stEIP3670/CreateTransactionEOF1ValidOpcodesFiller.yml

```diff
@@ -0,0 +1,1700 @@
+# Creation transaction deploying EOF code containing each valid Opcode
+---
+CreateTransactionEOF1ValidOpcodes:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data:
+      - ':label 00_stop            :raw 0x68ef000101000200000060005260096017f3'
+      - ':label 01_add             :raw 0x68ef000101000200010060005260096017f3'
```

Comment:
All opcodes need to have enough items on stack to pass 5450 stack validation

Reply:
Ant BTW for simplicity it can be the same max number of possible arguments to an opcode (is that 7 for CALL?) for every opcode, because it is allowed to have extra items on stack after termination.

But also let's use STOP instead of RETURN for termination, because it doesn't require stack arguments

Reply:
Added enough stack items at the beginning